library/systemd/README.md

```diff
@@ -0,0 +1,76 @@
+# Systemd actions
```

Comment:
This file should be visible in the YARD output, and linked from the relevant classes. Maybe it should be inlined in the relevant classes instead?

Reply:
Well, I can link the file in relevant classes, but can I make it part of YARD output?

Reply:
@joseivanlopez add it to .yardopts